src/Redistribution.sol

```diff
@@ -400,119 +396,80 @@ contract Redistribution is AccessControl, Pausable {
             inProximity(entryProof1.proveSegment, currentRevealRoundAnchor, winner.depth),
             "witness is not in depth"
         );
-        inclusionFunction(entryProof1, x * 2);
+        inclusionFunction(entryProof1, indexInRC1 * 2);
         stampFunction(entryProof1);
         socFunction(entryProofLast);
 
         require(
             inProximity(entryProof2.proveSegment, currentRevealRoundAnchor, winner.depth),
             "witness is not in depth"
         );
-        inclusionFunction(entryProof2, y * 2);
+        inclusionFunction(entryProof2, indexInRC2 * 2);
         stampFunction(entryProof2);
         socFunction(entryProofLast);
 
-        checkOrder(x, y, entryProof1.proofSegments[0], entryProof2.proofSegments[0], entryProofLast.proofSegments[0]);
+        checkOrder(
+            indexInRC1,
+            indexInRC2,
+            entryProof1.proofSegments[0],
+            entryProof2.proofSegments[0],
+            entryProofLast.proofSegments[0]
+        );
 
         emit WinnerSelected(winner);
-
         PostageContract.withdraw(winner.owner);
     }
 
-    function winnerSelection() internal returns (Reveal memory winner_) {
+    function winnerSelection() internal returns (Reveal memory _winner) {
```

Comment:
I don't think that kind of convention exists in Solidity (feel free to point me otherwise).
I have never seen trailing underscore used so far, it's even not common to use leading in return vars.